routers/api/v1/repo/issue.go

```diff
@@ -161,6 +161,12 @@ func EditIssue(ctx *context.APIContext, form api.EditIssueOption) {
 		ctx.Error(500, "UpdateIssue", err)
 		return
 	}
+	if form.State != nil {
+		if err = issue.ChangeStatus(ctx.User, ctx.Repo.Repository, strings.Compare("closed", *form.State) == 0); err != nil {
```

Comment:
Doesn't seem like I can do that...  
`invalid type assertion: gogs.STATE_CLOSED.(string) (non-interface type gogs.StateType on left) (build)at line 165 col 1`  
`cannot use gogs.STATE_CLOSED (type gogs.StateType) as type string in argument to strings.Compare (build)at line 165 col 1`